jt-all/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>jaitools</artifactId>
         <groupId>org.jaitools</groupId>
-        <version>1.5-SNAPSHOT</version>
+        <version>1.6-SNAPSHOT</version>

```

Comment:
Do we need to branch off 1.5.x?

Reply:
This change was just missed (since this module is not built when just deploying to maven).

So yes we should branch, and I should of done it earlier.

Reply:
I made sure the 1.5 tag has been pushed to upstream here